schema.cue

```diff
@@ -0,0 +1,62 @@
+import (
+	"time"
+)
+
+// Date is a date in the format YYYY-MM-DD
+#Timestamp: time.Format(time.RFC3339)
+
+#TypeURI: string
+
+#BaselineControlID: =~"^OSPS-[A-Z]{2}-[0-9]{2}.[0-9]{2}$"
+
+#ResourceDescriptor: {
```

Comment:
This should come from https://github.com/in-toto/attestation/blob/main/spec/v1/resource_descriptor.md